composer/workflows/airflow_db_cleanup.py

```diff
@@ -79,9 +83,10 @@
 DAG_OWNER_NAME = "operations"
 # List of email address to send email alerts to if this job fails
 ALERT_EMAIL_ADDRESSES = []
+# Airflow version used by the environment
+AIRFLOW_VERSION = airflow.version.version.split(".")
```

Comment:
I would do "from airflow.version import version" for better readability.

Reply:
Done.